### PR TITLE
Landing improvements (May 5)

### PR DESCRIPTION
**Pull request by kellan-simiotics**
## Changes

Text and style changes to landing page based on latest designs.

## How to test these changes?

Tested locally.

### CODE REVIEW

File: frontend/pages/index.js [hunk 1]
```diff
@@ -180,6 +182,62 @@ const Homepage = () => {
...
+          textAlign="center"
+        >
+          {headingText}
+        </Text>
+        <Text textAlign="center">{description}</Text>
+      </Box>
+    );
+  };
+
+  const JoinDiscordButton = () => {
```
Comment (peersky) [inline frontend/pages/index.js:207]: If you want to have reusable button styling then define it it in theme/button under a new `discordButton` variant, that way we will be able to reuse styling across other pages and components

File: frontend/pages/index.js [hunk 2]
```diff
@@ -180,6 +182,62 @@ const Homepage = () => {
...
+        fontWeight="bold"
+        rounded={["lg", "xl", "2xl"]}
+        shadow="md"
+        variant="solid"
+        colorScheme="orange"
+        textColor="blue.1200"
+        fontSize={["lg", "xl", "2xl", "3xl", "4xl", "4xl"]}
+        py={[4, 6, 6, 8, 8]}
+        px={[4, 4, 4, 8, 8]}
+        onClick={() => {
```
Comment (peersky) [inline frontend/pages/index.js:223]: OnClick should be realized in JoinDiscordButton implementation

File: frontend/pages/index.js [hunk 3]
```diff
@@ -180,6 +182,62 @@ const Homepage = () => {
...
+              buttonName: `Join our Discord`,
+              page: `landing`,
+              section: `bottom-line`,
+            });
+          }
+        }}
+        href={"/discordleed"}
+        isExternal
+      >
+        Join our Discord
```
Comment (peersky) [inline frontend/pages/index.js:236]: Button name should be realized in button implementation (passed here as props)

File: frontend/pages/index.js [hunk 4]
```diff
@@ -180,6 +182,62 @@ const Homepage = () => {
       setBackgroundLoaded3840(true);
     };
   }, []);
+
+  const MilestoneBox = ({ headingText, description }) => {
```
Comment (peersky) [inline frontend/pages/index.js:186]: Move this to components and export as `chakra(MilestoneBox)`

File: frontend/pages/index.js [hunk 5]
```diff
@@ -290,9 +327,31 @@ const Homepage = () => {
...
+                  />
+                  <MilestoneBox
+                    headingText="22,000 active users"
+                    description=""
+                  />
+                </Flex>
+                <Text textAlign="center" fontSize="lg">
+                  And growing...
+                </Text>
+                <chakra.h2
```
Comment (peersky) [inline frontend/pages/index.js:346]: use <Heading as="h2" ... > instead

File: frontend/pages/index.js [hunk 6]
```diff
@@ -381,111 +444,185 @@ const Homepage = () => {
...
+                    image={assets["lender"]}
                     altText="mined transactions"
-                    heading="CONTENT MANAGEMENT"
+                    heading="Airdrops"
                   />
                 </SimpleGrid>
               </GridItem>
               <GridItem
-                px="7%"
+                paddingLeft="15%"
```
Comment (peersky) [inline frontend/pages/index.js:484]: Paddings on left and right on grid item should be equal. Use ` px={["7%", null, "12%", "15%"]}` or `px="7%"` and solve any other required paddings by using css aligning/justifying, if that's not enough - You can add any additional paddings you need in child components of GridItem

Comment (peersky) [reply frontend/pages/index.js:484]: image

**Comment by peersky** [comment]
Can you center out this section on mobile screen and also reduce paddings on small screen size. Also add `minH="100vh"` for any `<GridItem>` that misses it
image

**Comment by peersky** [comment]
image
More padding between button and cards on mobile view

**Comment by peersky** [comment]
image
This section text is very small on large screen (3000px here)

**Comment by peersky** [comment]
image
Seems to be aligned with card in one line on smaller screen size but on >2560 becomes padded

**Comment by peersky** [comment]
In mockup text looks bigger and also subheader is bolder. Can you try using semibold or `weight` property on text there and try to make title text bigger ? 
image

**Comment by peersky** [comment]
Here also, see if you can make text weight a bit bolder
image

**Comment by peersky** [comment]
Can you make steps in bolder? Will look better

image

**Review by peersky — CHANGES_REQUESTED** [review]
changes requested

File: frontend/pages/index.js (hunk 2 above)
Comment (peersky) [reply frontend/pages/index.js:223]: Expected code would look like 
page
```jsx
<Button variant="DiscordButton" onClick={() => { mixpanel.track()}}>Join us in discord</Button>
```

That way also you can add mixpanel track property to identify from which section button was pressed

**Comment by kellan-simiotics** [comment]
@peersky @zomglings Please review changes.

**Review by zomglings — APPROVED** [review]
lgtm

**Pull request merged**